draft-spaghetti-sshm-filexfer.xml

```diff
@@ -94,7 +107,7 @@
 
     <t>
       Even though this protocol is described in the context of the SSH2 protocol, this protocol is general and independent of the rest of the SSH2 protocol suite.
```

Comment:
is this actually true? We now should have 20+ years of experience with this protocol, did it pop up anywhere else as far as we know?